lib/car.rb

```diff
@@ -0,0 +1,26 @@
+class Car
+
+  attr_accessor :make, :model, :monthly_payment, :loan_length, :color
+
+  def initialize(make_and_model, monthly_payment, loan_length)
+    @monthly_payment = monthly_payment
+    @loan_length = loan_length
+    make_and_model = {
```

Comment:
Nice thought here. This was tricky because the make and model was coming in as a single string.
``` 
"Ford Mustang"
```
So we would have wanted to split this into two pieces. One way to do this could be to create a method that does the splitting and then call that in the initialize. https://ruby-doc.org/core-2.4.0/String.html#:~:text=split(pattern%3Dnil,4%22%2C%20%22%22%2C%20%22%22%5D%0A%0A%22%22.split(%27%2C%27%2C%20-1)%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%23%3D%3E%20%5B%5D